app/helpers/groups_helper.rb

```diff
@@ -1,5 +1,10 @@
 module GroupsHelper
 
+
+  def set_current_group
```

Comment:
oh shoot, I forgot to delete this! we don't need this function, was just trying it out. will update the pull request.